projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -243,6 +250,15 @@ const pathsAreEqual = (a: PathToIssue, b: PathToIssue) =>
 const IssueScreenWithPath = React.memo(
     ({ path }: { path: PathToIssue }) => {
         const response = useIssueResponse(path)
+        const weatherVisibility = useWeatherVisibility()
+        if (weatherVisibility.loading) return null
+        let isWeatherShown = false
+        if (weatherVisibility.error) {
+            console.error(weatherVisibility.error)
```

Comment:
Are we sending these back to sentry or do we need to wrap them in something?

Reply:
No, I can replace by `throw` so it get caught by the closest error boundary, which is indeed kind of better.

Reply:
oooh, fancy